TranslatorGuidelines.md

```diff
@@ -3,18 +3,18 @@
 ## レッスンのタイトル
 
 最初に翻訳を始めた人が決めます。
-そのあとは統一しましょう。
+そのあとに統一しましょう。
 
 ## 本文
 
 「です・ます調」を基本とします。
 
-直訳よりも、分かりやすいかつ自然な日本語を優先します。
-したがって、言葉を逐一訳す必要もなく、文章のフレーズの順番を守る必要もありません。
+直訳よりも、分かりやすく自然な日本語を優先します。
+従って、言葉を逐一訳す必要もなく、文章のフレーズの順番を守る必要もありません。
 
-`.po`ファイルを編集している時に、`\n`で改行しているように見えますが、これは本当の改行ではありあません（`.md`ファイルと同じく、一行開けることによって改行が入る）。
+`.po`ファイルを編集している時に、`\n`で改行しているように見えますが、これは本当の改行ではありあません（`.md`ファイルと同じく、一行開けることによって改行が入ります）。
 なので、`\n`は無視して下さい（英語と日本語で`\n`の数・場所を合わせる必要はありません）。
-翻訳をする際、`\n`を入れると書きやすく・見やすくなるのであれば、入れても構いません。
+翻訳をする際、改行を入れると書きやすく・見やすくなるのであれば、改行(`\n`)を入れても構いません。
```

Comment:
The problem here is that `\n` is not a true line break. If the translator wants to include `\n` to make things easier to read, that's fine. But we don't want to confuse it with a true markdown line break (i.e., an open line, or two spaces at the end of a line).
```suggestion
翻訳をする際、改行を入れると書きやすく・見やすくなるのであれば、`\n`を入れても構いません。
```